src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -11,11 +11,14 @@
     <FileAlignment>512</FileAlignment>
     <DebugSymbols>true</DebugSymbols>
     <PlatformTarget>AnyCPU</PlatformTarget>
-    <DefineConstants>$(DefineConstants)</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)'=='netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>
+    <EnableWinRT Condition="'$(TargetGroup)'=='netcore50aot'">true</EnableWinRT>
```

Comment:
Correct. We should only keep this property if it is actually used in the project somewhere.